cmd/state/main.go

```diff
@@ -261,12 +260,21 @@ func run(args []string, isInteractive bool, cfg *config.Instance, out output.Out
 }
 
 func argsHaveVerbose(args []string) bool {
-	for _, arg := range args {
+	var commandFound, isRunOrExec bool
+	nextArg := 0
+
+	for i, arg := range args {
+		if i > 0 && !commandFound && !strings.HasPrefix(arg, "-") {
+			commandFound = true
+			isRunOrExec = arg == "run" || arg == "exec"
+			nextArg = i + 1
+		}
+
```

Comment:
I've not reasoned this concern fully, but... I think this will trip up on `state --output json`. I wonder if the spf13/cobra or spf13/pflag packages have helper functions we can leverage.

Reply:
You are right. I couldn't find any helper functions. If there were, there'd be a lot of repetition thanks to our `cmdtree.New()` structure. Perhaps this is one of those cases where "good enough" applies, rather than 100% correct.

Reply:
I hear you, but I disagree that this is good enough. Unless we document that double dash flags cannot use spaces and must use the equals sign. But that breaks POSIX, I think (I've not looked it up, but it seems like it would).

Reply:
As for taking a direction, please go wit this for now and file a bug for us to resolve handling this issue. It may be related to the other bug I've filed, but I suggest we decide that during triage.

Reply:
Sorry, what I mean by "good enough" was my most recent commit: https://github.com/ActiveState/cli/pull/1860/commits/6f9b08ce0f041aa3b31f5e4957f84cf7c469a76b

It does not trip up on `--option value` flags.

Reply:
I've filed a bug: https://activestatef.atlassian.net/browse/DX-961